__test__/pages/search/advancedSearchForm.test.tsx

```diff
@@ -32,32 +34,36 @@ describe("Advanced Search Form", () => {
   // final input in output string in test. the broken test is
   // commented out below.
   it.todo("can set keyword, contributor, title, subject")
-  // async () => {
-  //   render(<AdvancedSearch isAuthenticated={true}/>)
+  // , async () => {
+  //
 
   //   const [keywordInput, contributorInput, titleInput, subjectInput] = [
-  //     "Keywords",
+  //     "Keyword",
   //     "Title",
   //     "Author",
   //     "Subject",
+  //     "Call number",
+  //     "Unique identifier",
   //   ].map((field) => screen.getByLabelText(field))
-  //   await act(async () => {
-  //     await userEvent.type(subjectInput, "italian food")
-  //     await userEvent.type(keywordInput, "spaghetti")
-  //     await userEvent.type(contributorInput, "strega nonna")
-  //     await userEvent.type(titleInput, "il amore di pasta")
-  //     // this set stimeout is to ad
-  //     // eslint-disable-next-line @typescript-eslint/no-empty-function
-  //     setTimeout(() => {}, 300)
-  //     submit()
+  //   fireEvent.change(subjectInput, { target: { value: "italian food" } })
+  //   fireEvent.change(keywordInput, { target: { value: "spaghetti" } })
+  //   fireEvent.change(contributorInput, { target: { value: "strega nonna" } })
+  //   fireEvent.change(titleInput, { target: { value: "il amore di pasta" } })
+  //   submit()
+  //   await waitFor(() =>
```

Comment:
Why doesn't this work?

Reply:
I don't know! I've tried it as `fireEvent`, as `userEvents`, `userEvents` wrapped in an `act`. It has worked in the past when run alone with a `setTimeout`. I've spent so much time trying to get these to work over the past year. I'm not sure. I know it's not ideal, but it's functionality that is pretty obvious when broken, so I'm not too worried about it. I know that Emma wrote a bunch of tests for the text inputs on My Account, and I'd like to ask her to take a look at it with me.